scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -0,0 +1,61 @@
+version 1.0
+
+import "GvsUnified.wdl" as Unified
+
+workflow GvsVariantCalling {
```

Comment:
Is the idea that this is what we will use for the beta workspace?

Reply:
yes!